project_resource_calendar/i18n/fr.po

```diff
@@ -238,25 +248,40 @@ msgid "Description"
 msgstr "Description"
 
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_display_name
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:111
+#, python-format
+msgid "Discard"
+msgstr "Discard"
+
+#. module: project_resource_calendar
 #: model:ir.model.fields,field_description:project_resource_calendar.field_report_project_resource_calendar_weekly_report_view_display_name
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_instrument_display_name
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_miscellaneous_display_name
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_room_display_name
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_room_type_display_name
 #: model:ir.model.fields,field_description:project_resource_calendar.field_weekly_report_wizard_display_name
 msgid "Display Name"
-msgstr "Nom à afficher"
+msgstr "Nom affiché"
 
 #. module: project_resource_calendar
-#: model:ir.ui.view,arch_db:project_resource_calendar.view_reservation_double_booking_validation_wizard
-msgid "Double Booking Validation"
-msgstr "Validation de la double réservation"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:172
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:59
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:114
+#: code:addons/project_resource_calendar/static/src/js/update_event_wizard.js:47
+#: code:addons/project_resource_calendar/static/src/js/update_event_wizard.js:86
+#, python-format
+msgid "Do you want to continue?"
+msgstr "Voulez vous continuer ?"
```

Comment:
```suggestion
msgstr "Voulez vous continuer?"
```

Reply:
A space is needed before and after the question mark in French.